w3-filecoin.md

```diff
@@ -28,47 +28,47 @@ There are several roles in the authorization flow:
 
 | Name          | Description |
 | ------------- | ----------- |
-| Storefront    | [Principal] identified by [`did:web`] identifier, representing a storage API like web3.storage |
-| Aggregator    | [Principal] identified by `did:key` identifier, representing a storage aggregator like w3filecoin |
-| Dealer        | [Principal] identified by `did:key` identifier that arranges filecoin deals with storage providers. e.g. Spade |
-| Chain Tracker | [Principal] identified by `did:key` identifier that tracks the filecoin chain |
+| Storefront    | [Principal] identified by a DID, representing a storage API like web3.storage |
+| Aggregator    | [Principal] identified by a DID, representing a storage aggregator like w3filecoin |
+| Dealer        | [Principal] identified by a DID, that arranges filecoin deals with storage providers. e.g. Spade |
+| Deal Tracker  | [Principal] identified by a DID, that tracks deals made by the Dealer |
 
 ### Storefront
 
-A _Storefront_ is a type of [principal] identified by a [`did:web`] identifier.
+A _Storefront_ is a type of [principal] identified by a DID (typically a [`did:web`] identifier).
```

Comment:
Can we give a reason for this?

Reply:
Intention is to allow storefront perform a key rotation without coordinating it with other actors in the system. That said I think it would make more sense to say

```suggestion
A _Storefront_ is a type of [principal] identified by a DID. It is RECOMMENDED to use  [`did:web`] identifier so that underlaying keys could be rotated without coordinating with other actors. 
```